src/routes/+page.server.ts

```diff
@@ -63,7 +61,9 @@ export const actions: Actions = {
         '&nbsp;  \n' +
         attachments +
         '&nbsp;  \n' +
-        `${getPriorityLabel(priority)} priority ${type} reported by ${author}`;
+        `${getPriorityLabel(
+          priority as priorityType
```

Comment:
Sorry, I would avoid using the `as`.
In a last resort, I would change the function input to
```
`${priorityType}` | string
```

What do you think?

Reply:
Feel free to merge it after reviewing this 🙏